.travis.yml

```diff
@@ -0,0 +1,26 @@
+jdk:
+  - oraclejdk8 # openJDK crashes sometimes; tensorframes is compiled w.r.t JDK8
```

Comment:
We are using TensorFlow and TensorFrames compiled with Java 8. Thus using JDK8 here.